BLSQ/iaso
# Wc2 93 csv/xls exports of logs

Add export options to the API (both XLSX and CSV) for storage logs per device + also merged the work of Christophe to add the buttons directly in the UI.

## Self proof reading checklist


## How to test

Play with the `/api/storage/<DEVICE_TYPE>/<DEVICE_ID>/logs?csv=true` and `/api/storage/<DEVICE_TYPE>/<DEVICE_ID>/logs?xlsx=true` and check if the results make sense.

**Update**: @beygorghor frontend code was merged, so the feature can be tested directly in the web UI: go to the details of a storage device, make sure there are logs displayed for this device, click on "CSV" and "XLSX" buttons and check the results.

## Notes

This PR is based on WC2-92, so the code would be shorter to review if this is done after merging WC2-92 into main. **Update: done, WC2-92 was merged this morning**.
